lib/Sabesp.js

```diff
@@ -24,7 +24,7 @@
 
       request({
           'url': url,
-          'method': 'POST',
+          'method': 'GET',
```

Comment:
Expected indentation of 8 spaces but found 10.

[Read more about it here.](http://eslint.org/docs/rules/indent)